.github/workflows/e2e_test_release_install.yml

```diff
@@ -27,8 +27,15 @@ jobs:
             runner: macos-12
             target_os: darwin
             target_arch: x86_64
+          - name: Windows x64
+            runner: rust-windows-x64
```

Comment:
@sgrebnov that is our hosted runner, we shouldn't install spice there, let's use `windows-latest` instead